lib/manager/npm/post-update/pnpm.spec.ts

```diff
@@ -23,56 +17,60 @@ describe('manager/npm/post-update/pnpm', () => {
     config = { cacheDir: 'some-cache-dir', constraints: { pnpm: '^2.0.0' } };
     env.getChildProcessEnv.mockReturnValue(envMock.basic);
   });
+
   it('generates lock files', async () => {
     const execSnapshots = mockExecAll(exec);
-    fs.readFile = jest.fn(() => 'package-lock-contents') as never;
+    fs.readLocalFile = jest.fn(() => 'package-lock-contents') as never;
```

Comment:
```suggestion
    fs.readLocalFile.mockResolvedValueOnce( 'package-lock-contents');
```
Please fix this file too